spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -249,38 +249,41 @@ def deactivate_profile_update_enrollment(status:)
       end
 
       # To be completed in a future ticket
-      # context 'User fails IPP and fails TMX review' do
```

Comment:
Except for some commented-out `binding.pry`s, all I did here was uncomment what was here.